src/lib/__tests__/RoutingManager-test.ts

```diff
@@ -290,8 +290,7 @@ describe('RoutingManager', () => {
       const fakeSearchBox: any = createFakeSearchBox();
       const fakeHitsPerPage = createFakeHitsPerPage();
 
-      search.addWidget(fakeSearchBox);
-      search.addWidget(fakeHitsPerPage);
+      search.addWidgets([fakeSearchBox, fakeHitsPerPage]);
```

Comment:
Did the codemod also merge these two lines or did you do it manually?

```ts
// Did it transform it to this?
search.addWidgets([fakeSearchBox]);
search.addWidgets([fakeHitsPerPage]);

// Or this?
search.addWidgets([fakeSearchBox, fakeHitsPerPage]);
```

Reply:
This was done manually, I did some touching up to avoid duplicates

Reply:
Are there ways to merge successive statements?

Reply:
Probably, but i couldn’t find a simple way to do that